PyZAPI/zappa_settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "default": {
+        "app_function": "pyzapi.app.zappa_dev",
+        "aws_region": "us-east-1",
+        "profile_name": "default",
+        "s3_bucket": "zappa-0oqiolisl",
```

Comment:
Agreed, sorry this one slipped through will create the example file and push now